src/components/SplashScreen.js

```diff
@@ -1,62 +1,64 @@
-import logo from '../assets/img/teamMX-logo.png';
-import '../assets/css/style.css';
-import {ClipLoader} from 'react-spinners';
-import { Button, ButtonGroup, Slide, Typography } from '@mui/material';
+import { Box } from "@mui/system";
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import logo from "../assets/img/teamMX-logo.png";
+import "../assets/css/style.css";
+import { ClipLoader } from "react-spinners";
+import { Button, ButtonGroup, Slide, Typography } from "@mui/material";
 
 export default function SplashScreen(props) {
-    const [loading, setLoading] = useState(false)
-    const navigate = useNavigate()
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate();
 
-    useEffect(() => {
-        setLoading(true)
-        setTimeout(() => {
-            setLoading(false)
-        }, 2000)
-    }, [])
+  useEffect(() => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+    }, 2000);
+  }, []);
 
-    return (
-        <Box sx={{
-            display: 'flex',
-            flexDirection: 'column',
-            width: '100%',
-            height: '100vh',
-            background: 'linear-gradient(180deg, #e20e2d, #aa0f26)'
-        }}>
-            <div className='splashBG'>
-                <div className='logo'>
-                    <img className='image' src={logo} alt='Team Logo'/>
-                </div>
-                <div>
-                    {
-                        loading ?
-                            // While loading show loader image from react-spinners
-                            <div className='sweet-loading'>
-                                <ClipLoader color='#edc314' loading={loading} size={30} />
-                            </div>
-                        :
-                            //After loading is complete, show buttons
-                            <Slide direction='up' in={!loading}>
-                                <ButtonGroup orientation='vertical'>
-                                    <Button 
-                                        variant='contained'
-                                        color='primary'
-                                        sx={{m:2, width:300,height:50}}
-                                        onClick={() => {
-                                            navigate('/login')
-                                        }}
-                                    >
-                                        <Typography variant='h6'>
-                                            Sign In
-                                        </Typography>
-                                    </Button>
-                                </ButtonGroup>
-                            </Slide>
-                    }
-                </div>
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        width: "100%",
+        height: "100vh",
```

Comment:
FYI this will might look weird on mobile - safari considers the browser toolbar to be a part of the viewport, so when you scroll on this page the component will seem like it's resizing randomly (~5vh worth)

Reply:
In this case do I need to change the height to percent?